packages/components/src/Chip/Chip.tsx

```diff
@@ -9,7 +9,7 @@ import { useChildComponent } from "./hooks/useChildComponent";
 import { Typography } from "../Typography";
 import { Tooltip } from "../Tooltip";
 
-export const Chip = ({
+export function Chip({
```

Comment:
I thought this was a possible cause, it's not.

I'm fine to revert it, but also this is more our style for function definitions in general.